DOCUMENTATION.md

```diff
@@ -0,0 +1,30 @@
+This document describes our rules for documenting things
+
+## The Rules
+
+1. Remember that all our documentation is public-facing.
+2. All engineering documentation in its final state must live in GitHub.
```

Comment:
I'll clarify this line to "code (python, js) and configuration (terraform) development" 

Security docs and operational docs are their own whole different category of things. Security docs shouldn't follow these rules at all. Operational docs are hit / miss.  

Runbooks specifically should follow the "redact IDs, use generic URLs" (or however I say it, I'm on mobile right now) rules below, so I don't personally see a strong reason not to co-locate them with the majority of the other documentation. Moreso because I'm going to be the primary author of runbooks, and I don't personally feel compelled to push them into an internal wiki.

This speaks more to the repository organization than anything else, and that subject is WIP. Anyone due to be eventually on call (which includes me for sure, and idk who beyond that) should make this call specifically. So idk maybe @chouinar has an opinion on where runbooks should live, or maybe he never wants to be on call 😆

Reply:
I think having a dedicated folder in our documentation for runbooks might help? Even if some of the sections just link to other parts of the docs.

Especially in an oncall critical situation, I want a single place to start.